charts/activemq-artemis/values.yaml

```diff
@@ -11,8 +11,12 @@ imageTag: latest
 
 ## Create a database user
 ##
-artemisUser: artemis
-artemisPassword: simetraehcapa
+artemis:
+  useDefaultUser: true
+  defaultUser:
+    artemisUser: artemis
```

Comment:
If we are using the defaultuser, why the need to specify the username here and not just in the secret?